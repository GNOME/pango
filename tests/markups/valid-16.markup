<span size="larger">larger and <span size="smaller">smaller <span size="smaller">and smaller</span></span> <span size="12800"><span size="larger">now with base</span></span></span>
