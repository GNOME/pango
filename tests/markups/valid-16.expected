larger and smaller and smaller now with base


---

range 0 11
0 44 scale 1.200000
range 11 19
11 30 scale 1.000000
range 19 30
19 30 scale 0.833333
range 30 31
0 44 scale 1.200000
range 31 44
0 44 scale 1.200000
31 44 size 15360
range 44 2147483647


---

[0:11] (null) Normal 0
[11:19] (null) Normal 0
[19:30] (null) Normal 0
[30:31] (null) Normal 0
[31:44] (null) Normal 18
[44:2147483647] (null) Normal 18
